controllers/webhook/storageclass.go

```diff
@@ -0,0 +1,44 @@
+package webhook
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"slices"
+
+	"github.com/go-logr/logr"
+	"github.com/red-hat-storage/ocs-operator/v4/controllers/util"
+	storagev1 "k8s.io/api/storage/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+type StorageClassAdmission struct {
+	client.Client
+	Decoder admission.Decoder
+	Log     logr.Logger
+}
+
+var supportedProvisioners = []string{
+	util.RbdDriverName,
+	util.CephFSDriverName,
```

Comment:
nfs is missing if its enabled.

Reply:
a separate PR should follow for NFS, I can't send PR for something I didn't try out.